upstream/solr6/local/92-init-solr.sh

```diff
@@ -152,10 +152,10 @@ function makeConfigs {
         setOption 'solr.suggester.enabled' "$ALFRESCO_SOLR_SUGGESTER_ENABLED" "$CONFIG_FILE_CORE"
         # SOLR_BACKUP_DIR only useful for ASS v2.0.2* and up
         if [ $coreAlfrescoName != $coreName ]; then
-          setOption 'solr.backup.dir' "$SOLR_BACKUP_DIR/$coreName/$coreAlfrescoName" "$CONFIG_FILE_CORE"
+          setOption 'solr.backup.dir' "$SOLR_BACKUP_DIR/$coreName" "$CONFIG_FILE_CORE"
```

Comment:
We must also validate if this will still work for sharded setups. LMK if you need help with that